### DISCUSSION TITLE
Any way to force landscape orientation for an app ?

### DISCUSSION THREAD

**Discussion by jtraulle** [PlayCover/PlayCover#718] [Q&A]
Hello,

I am using an app that have different layout for portrait mode and landscape mode.

https://apps.apple.com/fr/app/smartrow/id1223542275

When used in portrait mode, the layout is similar to this :
![230x0w](https://user-images.githubusercontent.com/613615/213175323-6928e213-38a1-42f3-8ba5-ca82088ca156.jpeg)

But, when used in landscape mode, the layout is similar to this :
![screen-7](https://user-images.githubusercontent.com/613615/213175415-74145ac7-6709-446a-b9e3-cdaae264ad24.jpeg)

There is no way in the app to trigger the change of orientation (no button), this is done only based on the detection (accelerometer).

Is there any way in PlayCover to trigger a change of orientation or force the orientation mode ?

I searched but does not found anything, the app always remains in portrait mode (default I guess) on PlayCover.

Thanks for your help :)

**Comment by rayzliu** [comment]
I have the same problem, some apps are designed for portrait mode only and I cannot even press "I accept EULA" in landscape mode, basically makes the app unusable
